service/src/test/java/uk/gov/hmcts/reform/fpl/service/PbaNumberServiceTest.java

```diff
@@ -54,6 +55,51 @@ void shouldUpdateForC2DocumentBundle() {
         assertThat(updatedC2DocumentBundle).isEqualTo(expectedUpdatedDocumentBundle);
     }
 
+    @Test
+    void shouldUpdatePbaNumberToIncludePrefix() {
+        String pbaNumber = "1234567";
+
+        String updatedPbaNumber = pbaNumberService.update(pbaNumber);
+
+        assertThat(updatedPbaNumber).isEqualTo("PBA1234567");
+    }
+
+    @Test
+    void shouldReturnNullWhenEmptyPbaNumber() {
+        String pbaNumber = "";
+
+        String updatedPbaNumber = pbaNumberService.update(pbaNumber);
+
+        assertNull(updatedPbaNumber);
```

Comment:
for consistency would be better to use same library everyhwere if possible
assertThat().isNull()